packages/analytics-browser/src/browser-client.ts

```diff
@@ -81,6 +85,14 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
     await this.add(new Context()).promise;
     await this.add(new IdentityEventSender()).promise;
 
+    if (isFileDownloadTrackingEnabled(this.config.autoTracking)) {
+      await this.add(fileDownloadTracking()).promise;
+    }
+
+    if (isFormInteractionTrackingEnabled(this.config.autoTracking)) {
```

Comment:
Minor, for page view, do we plan to make it similar like this to conditionally add the plugin or not?

Reply:
For page views, I used `trackOn` config as toggle, although that means we will install it all the time. These two new plugins do not have options defined yet so the condition was added in browser-client level.

Reply:
Any thoughts?

Reply:
I am good with the `trackOn`. If the `trackOn` config is used with some custom function, do we want to merge the logic there? I think currently it will be assigned with `attribution` value only. Example:
```
init(API_KEY, 'user@amplitude.com', {
  pageViewTracking: {
    trackOn: () => {
      return window.location.pathname === '/landing_page'
    },
  }
});
```

Reply:
Yeah this makes sense. Although I was under the impression this is out of scope and we want to have a boolean config first. I can check with others.

Reply:
@liuyang1520 the advanced controls are now added :)